src/nidigital/templates/session.py/fancy_fetch_history_ram_cycle_information.py.mako

```diff
@@ -30,6 +30,8 @@
                 'position: Specified value = {0}, samples_to_read: Specified value = {1}; Samples available = {2}.'
                 .format(position, samples_to_read, samples_available - position))
 
+        # Site can be 'N', N or 'siteN'. This will normalize all options to 'siteN' which is requried by the driver
+        site = _converters.convert_site_string(site)
```

Comment:
Why not modify the private functions to use the repeated capability?
Sure, they are private, so it doesn't affect the Python client. But it does mean that there is an internal integrity problem of the metadata not being correct. We could want to reuse this metadata in other places after all.

Reply:
Because it is not a repeated capability. It is a single site, which is what the converter is for.